ios/Podfile.lock

```diff
@@ -188,33 +188,33 @@ PODS:
     - GoogleUtilities/Network (~> 7.4)
```

Comment:
We only need to update lottie related frameworks. Revert all other changes which are out of scope.

Just replace with this file I generated:

[Podfile.lock.zip](https://github.com/Expensify/App/files/12840514/Podfile.lock.zip)